def log_error(scope, e): print(f'[ERR][{scope}]', e)
